Notebooks/Russian Twitter Corpus/DataAnalys.py

```diff
@@ -0,0 +1,51 @@
+import csv
```

Comment:
Move this file into `Utils` folder and set for its name non-camel-case style :)

Reply:
If notebook and file are in different directories, how can I import from file?